southpark-search/README.md

```diff
@@ -304,17 +306,18 @@ The `callback` argument is used to post-process the returned message. In this de
 
 ```python
 ppr = lambda x: print_topk(x, text)
-fl.search(read_query_data(text), callback=ppr, topk=top_k)
+f.search_lines(lines=[text, ], output_fn=ppr, topk=top_k)
 ```
 
 ```python
 def print_topk(resp, word):
     for d in resp.search.docs:
-        for idx, kk in enumerate(d.topk_results):
-            score = kk.score.value
+        print(f'Ta-Dah🔮, here are what we found for: {word}')
+        for idx, match in enumerate(d.matches):
+            score = match.score.value
             if score < 0.0:
                 continue
-            doc = kk.match_doc.text
+            doc = match.meta_info.decode()
```

Comment:
are we sure about this? It should be extracted to the text of the `match` document right?

Reply:
@JoanFM I debug this one, the text attribute do not have a `[SEP]` mark to be splitted (please check the following line of code).

Reply:
Yes this has problem, I have fixed it